CONTRIBUTING.rst

```diff
@@ -99,6 +100,29 @@ If you are new to GitHub, below you can find a list of useful tutorials:
 - http://rogerdudler.github.io/git-guide/index.html
 - https://www.atlassian.com/git/tutorials
 
+Install pre-commit hooks
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you cloned the repository and the development environment is installed (and active),
+install the git pre-commit hook by executing the following command in the repository's
+root::
+
+    pre-commit hook install
+
```

Comment:
This seems like a useful addition! How did you run Black to set it to a line length of 88? --> Good to add this here too, to make sure everyone does it that way. :)

Reply:
Good point. The black configuration is stored in the `pyproject.toml` file. When black is run in any pysteps folder, it loads that configuration file. Since the line-length was not specified, its default value is used (88).  The pre-commit application only installs black, but with no associated configuration. 

I updated the developers' guidelines briefly explaining this and made the line-length explicit in the `pyproject.toml` file.